shapely/_geometry.py

```diff
@@ -435,10 +451,17 @@ def get_interior_ring(geometry, index, **kwargs):
 
     Examples
     --------
-    >>> polygon_with_hole = Geometry("POLYGON((0 0, 0 10, 10 10, 10 0, 0 0), (2 2, 2 4, 4 4, 4 2, 2 2))")
+    >>> from shapely import Point, Polygon
+    >>> polygon_with_hole = Polygon(((0, 0), (0, 10), (10, 10), (10, 0), (0, 0)), \
+holes=[((2, 2), (2, 4), (4, 4), (4, 2), (2, 2))])
```

Comment:
```suggestion
    >>> polygon_with_hole = Polygon(
    ...     [(0, 0), (0, 10), (10, 10), (10, 0), (0, 0)],
    ...     holes=[[(2, 2), (2, 4), (4, 4), (4, 2), (2, 2)]]
    ... )
```

Possible alternative, that might be more readable? (although it takes quite a bit more vertical space)